js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,13 @@
+function aperture(tuple, array) {
+  if (array.length < tuple) return [];
+  if (tuple == 0) return array;
+  let aperture = [];
+  for (let i = 0; i < (array.length - (tuple - 1)); i++) {
+    aperture.push(array.slice(i, i + tuple));
```

Comment:
Here we are creating multiple tuples, so according to me it would have been good if i can create a method to create a single tuple whose input would be indexToStartFrom or something similar, it would make the code atomi.